modules/rules_swift/3.0.2/MODULE.bazel

```diff
@@ -0,0 +1,66 @@
+module(
+    name = "rules_swift",
+    version = "3.0.2",
+    bazel_compatibility = [">=7.0.0"],
+    compatibility_level = 3,
```

Comment:
Are we sure we want to bump the compatibility level for rules_swift? This might break some users if rules_swift with different compatibility levels show up in the dependency graph.

@brentleyjones @keith @luispadron

Reply:
Yes this was intentional as a lot of the rules_swift public API has changed. We expect to have to update `max_compatibility_level` for a few of the bzlmod projects in the registry.

Reply:
Yeah, this version has lots of breaking changes. `max_compatibility_level` will cover most uses out there, but some will have to bump there minimum supported version.

Reply:
Thanks for confirming!